### ISSUE TITLE
i278 more macros

### ISSUE THREAD

**Issue by ckormanyos** [boostorg/math#304]
The macros in the list below use leading underscore.
In these macros, should the sub-string "_BOOST" be changed to sth like "BOOST_MATH"?
(Which then I could do myself if it should be done.)

```
#ifndef _BOOST_CSTDFLOAT_CMATH_2014_02_15_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_cmath.hpp	12	9
#define _BOOST_CSTDFLOAT_CMATH_2014_02_15_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_cmath.hpp	13	9
#endif // _BOOST_CSTDFLOAT_CMATH_2014_02_15_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_cmath.hpp	1093	11
#ifndef _BOOST_CSTDFLOAT_COMPLEX_2014_02_15_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_complex.hpp	12	9
  #define _BOOST_CSTDFLOAT_COMPLEX_2014_02_15_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_complex.hpp	13	11
#endif // _BOOST_CSTDFLOAT_COMPLEX_2014_02_15_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_complex.hpp	38	11
#ifndef _BOOST_CSTDFLOAT_COMPLEX_STD_2014_02_15_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_complex_std.hpp	13	9
  #define _BOOST_CSTDFLOAT_COMPLEX_STD_2014_02_15_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_complex_std.hpp	14	11
#endif // _BOOST_CSTDFLOAT_COMPLEX_STD_2014_02_15_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_complex_std.hpp	641	11
#ifndef _BOOST_CSTDFLOAT_IOSTREAM_2014_02_15_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_iostream.hpp	12	9
  #define _BOOST_CSTDFLOAT_IOSTREAM_2014_02_15_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_iostream.hpp	13	11
#endif // _BOOST_CSTDFLOAT_IOSTREAM_2014_02_15_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_iostream.hpp	774	11
#ifndef _BOOST_CSTDFLOAT_LIMITS_2014_01_09_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_limits.hpp	12	9
  #define _BOOST_CSTDFLOAT_LIMITS_2014_01_09_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_limits.hpp	13	11
#endif // _BOOST_CSTDFLOAT_LIMITS_2014_01_09_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_limits.hpp	85	11
#ifndef _BOOST_CSTDFLOAT_TYPES_2014_01_09_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_types.hpp	12	9
  #define _BOOST_CSTDFLOAT_TYPES_2014_01_09_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_types.hpp	13	11
#endif // _BOOST_CSTDFLOAT_BASE_TYPES_2014_01_09_HPP_	.\boost_1_72_0\boost\math\cstdfloat\cstdfloat_types.hpp	441	11
```

**Comment by NAThompson** [comment]
Aren't a lot of these macros around dead code?

**Comment by ckormanyos** [comment]
> Aren't a lot of these macros around dead code?

Good question.
I don't really know if anyone is using cstdfloat?
A couple test files were using them in the Boost distro.
But I do not know if any of world is using them

**Comment by ckormanyos** [comment]
Closed with PR #305
